### DISCUSSION TITLE
A question about Cubers.io results

### DISCUSSION THREAD

**Discussion by JudeStradtner** [euphwes/cubers.io#203] [General]
Hello, I was wondering if you could double-check the times put in by "Thatcubingkid"
I don't have a Reddit account, so I thought this would be the second-best place to ask about this.
He's only been competing for a few weeks but has achieved world-class times, and according to his Reddit bio, he averages sub-20 on 3x3.
https://www.reddit.com/user/Thatcubingkid/

**Comment by Avery-Levin** [comment]
I'll also add that he is a chronic liar on the WCA forums and only "corrected" his results when he was called out. He only corrected 3x3 because he is only competing in 3x3 at his upcoming competition and still makes dubious times. Overall, he is a cheater beyond a reasonable doubt.

**Reply by euphwes** [reply]
Hi, thanks for letting me know! I'll take a look at those results (and past ones) in a few hours, and remove the obviously bogus ones. I have a few volunteer moderators who verify high-placing participants but it's hard to keep an eye on all results.

**Comment by euphwes** [comment]
Thanks @JudeStradtner ! I took a pass through his times and his Reddit profile, and agree they are not plausible. I've hidden all of his results, and any future ones he posts won't be considered in overall rankings.
